notebooks/2_PrincipalCovariatesRegression.ipynb

```diff
@@ -83,23 +83,30 @@
     "(1-\\alpha){\\left\\lVert \\mathbf{Y} - \\mathbf{X}\\mathbf{P}_{XT}\\mathbf{P}_{TY}\\right\\rVert^2}.\n",
     "\\end{equation}\n",
     "\n",
-    "It is easier to minimize this by looking for a projection $\\tilde{\\mathbf{T}}$ in a latent space for which we enforce orthonormality, $\\tilde{\\mathbf{T}}^T\\tilde{\\mathbf{T}} = \\mathbf{I}$\n",
+    "It is easier to minimize this by looking for a projection $\\tilde{\\mathbf{T}}$ in a latent space if we enforce the column vectors in $\\tilde{\\mathbf{T}}$ to be orthonormal, i.e., $\\tilde{\\mathbf{T}}^T\\tilde{\\mathbf{T}} = \\mathbf{I}$. \n",
     "\n",
     "\\begin{equation}\n",
     "\\ell =\n",
-    "\\alpha {\\left\\lVert \\mathbf{X} - \\mathbf{X}\\mathbf{P}_{XV}\\mathbf{P}_{VX}\\right\\rVert^2}\n",
+    "\\alpha {\\left\\lVert \\mathbf{X} - \\mathbf{X}\\mathbf{P}_{X\\tilde{T}}\\mathbf{P}_{\\tilde{T}X}\\right\\rVert^2}\n",
     " +\n",
-    "(1-\\alpha){\\left\\lVert \\mathbf{Y} - \\mathbf{X}\\mathbf{P}_{XV}\\mathbf{P}_{VY}\\right\\rVert^2}.\n",
+    "(1-\\alpha){\\left\\lVert \\mathbf{Y} - \\mathbf{X}\\mathbf{P}_{X\\tilde{T}}\\mathbf{P}_{\\tilde{T}Y}\\right\\rVert^2}.\n",
     "\\end{equation}\n",
     "\n",
-    "$\\tilde{\\mathbf{T}}$ is the [whitened](https://en.wikipedia.org/wiki/Whitening_transformation) version of our earlier projection $\\mathbf{T}$."
+    "$\\tilde{\\mathbf{T}}$ is the [whitened](https://en.wikipedia.org/wiki/Whitening_transformation) version of our earlier projection $\\mathbf{T}$.\n",
+    "Recall what we saw in [MDS](1_LinearMethods.ipynb).\n",
+    "\n",
+    "\\begin{align}\n",
+    "& \\quad \\mathbf{K} = \\mathbf{X}\\mathbf{X}^T = \\mathbf{U}_\\mathbf{K} \\mathbf{\\Lambda}_\\mathbf{K} \\mathbf{U}_\\mathbf{K}^T \\\\\n",
+    "& \\quad\\mathbf{T} = \\hat{\\mathbf{U}}_\\mathbf{K} \\hat{\\mathbf{\\Lambda}}_\\mathbf{K}^{1/2} \\\\\n",
+    "& \\quad \\tilde{\\mathbf{T}} = \\hat{\\mathbf{U}}_\\mathbf{K}\n",
+    "\\end{align}\n"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "By definition, $\\mathbf{X}\\mathbf{P}_{X\\tilde{T}} = \\tilde{\\mathbf{T}}$. Due to orthogonality, the definition $\\tilde{\\mathbf{T}}\\mathbf{P}_{\\tilde{T}X} = \\mathbf{X}$ implies that $\\mathbf{P}_{\\tilde{T}X} = \\tilde{\\mathbf{T}}^T\n",
+    "By definition, $\\mathbf{X}\\mathbf{P}_{X\\tilde{T}} = \\tilde{\\mathbf{T}}$. Due to orthonormality, $\\tilde{\\mathbf{T}}^T\\tilde{\\mathbf{T}} = \\mathbf{I}$, the definition $\\tilde{\\mathbf{T}}\\mathbf{P}_{\\tilde{T}X} = \\mathbf{X}$ implies that $\\mathbf{P}_{\\tilde{T}X} = \\tilde{\\mathbf{T}}^T\n",
```

Comment:
```suggestion
    "By definition, $\\mathbf{X}\\mathbf{P}_{X\\tilde{T}} = \\tilde{\\mathbf{T}}$. Due to orthonormality, the definition $\\tilde{\\mathbf{T}}\\mathbf{P}_{\\tilde{T}X} = \\mathbf{X}$ implies that $\\mathbf{P}_{\\tilde{T}X} = \\tilde{\\mathbf{T}}^T\n",
```

We _just_ said the equation with respect to orthonormality. Repeating it here is unnecessary and will be confusing for the reader.